calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -640,7 +640,19 @@ private function getPageWithDetails($intPageId)
 	{
 		if (!isset(self::$arrPageCache[$intPageId]))
 		{
-			self::$arrPageCache[$intPageId] = PageModel::findWithDetails($intPageId);
+			$objPage = PageModel::findWithDetails($intPageId);
```

Comment:
Can `$objPage` be `null` as well?

Reply:
In case it was unclear where I was going with my question: We should add an early return if `$objPage` is `null` then.